FirebaseDynamicLinks/Sources/FIRDynamicLinks.m

```diff
@@ -440,7 +440,11 @@ - (nullable FIRDynamicLink *)dynamicLinkFromCustomSchemeURL:(NSURL *)url {
       }
     }
   }
-  mainQueueCompletion(nil, nil);
+  mainQueueCompletion(
+      nil, [NSError errorWithDomain:@""
+                               code:0
+                           userInfo:[NSDictionary dictionaryWithObject:@"Failed to parse!"
```

Comment:
"Universal link URL could not be parsed by Dynamic Links."

Reply:
Done